modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/generalized_plane_strain.md

```diff
@@ -79,9 +83,7 @@ N_{z} = \int_{A}{\sigma_{zz}dA} = \bar{N}_{z}
 \end{equation}
 The stress resultant $N_{z}$ conjugates with constant strain $\epsilon_{zz}$.
 
-The formulation above for generalized plane strain problem shares many similarities with that of the conventional plane stress or plane strain problem. +All the differences are associated with the introduction of an additional degree of freedom for the out-of-plane direction+.
-
-## Implementation
+## $x$-$y$ plane implementation
 
 The out-of-plane strain is a scalar variable, and it can be added to the standard system of equations for a mechanics problem, where $\boldsymbol{u}_x$ and $\boldsymbol{u}_y$ represent
```

Comment:
Rather than changing the stuff up above, I'd just inject something here at the end of this first sentence:

... represent the corresponding reaction forces. The discussion here is for the case where the two-dimensional model lies in the $x$-$y$ plane, which is typically the case. Modifications for cases where it lies in other planes are discussed later.